src/rewriter.ts

```diff
@@ -157,6 +185,36 @@ export abstract class Rewriter {
     this.position.position += str.length;
   }
 
+  sourcePositionOfNodeStart(node: ts.Node): SourcePosition {
+    const nodeStart = node.getStart();
+    const pos = this.file.getLineAndCharacterOfPosition(node.getStart());
+    return this.findOffsetOfFirstNonWhitespaceCharacter(
+        node.getText(), {line: pos.line, column: pos.character, position: nodeStart});
+  }
+
+  /**
+   * If you pass a pos, it returns a Source Position offset from that pos.
+   *  Otherwise, it just returns the absolute offsets
+   */
+  findOffsetOfFirstNonWhitespaceCharacter(str: string, pos = {line: 0, column: 0, position: 0}):
+      SourcePosition {
+    let {line, column, position} = pos;
+    for (const c of str) {
```

Comment:
Once we removed the old codepath / `SourceMapper` (as I mentioned in another comment), we don't need to care about row / column anymore, so this code can be simplified to just adjust the position, maybe via single regex.